lib/bolt/transport/sudoable/connection_helper.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'bolt/transport/sudoable/file_commands'
+
+module Bolt
+  module Transport
+    class Sudoable < Base
+      class ConnectionHelper
```

Comment:
Nitpick I think helpers are more idiomatically mixin modules. This seems like it just a base class so we can just call it Connection